plugins/portal-history-ornaments.js

```diff
@@ -0,0 +1,234 @@
+// @author         Johtaja
+// @name           Portal History Ornaments
+// @category       Highlighter
+// @version        0.1.0
+// @description    Display portal history as ornaments
+
+
+// use own namespace for plugin
+var portalsHistory = {};
+window.plugin.portalHistoryOrnaments = portalsHistory;
+
+// Exposed functions
+portalsHistory.toggleHistory        = toggleHistory;        // needed for button
+portalsHistory.toggleDisplayMode    = toggleDisplayMode;    // used by dialog
+portalsHistory.drawAllFlags         = drawAllFlags;         // hooked to 'mapDataRefreshEnd'
+
+var KEY_SETTINGS = "plugin-portal-history-flags";
+
+//------------------------------------------------------------------------------------------
+// Toggle Switch
+
+function makeButton () {
+  var isClass = portalsHistory.settings.drawMissing ? 'revHistory' : 'normHistory';
+  
+  $('.leaflet-top.leaflet-left').append(
+    $('<div>', { id: 'toggleHistoryButton', class: 'leaflet-control leaflet-bar' }).append(
+      $('<a>', {
+        id: 'toggleHistory',
+        title: 'History toggle',
+        class: isClass,
+        click: function () { toggleHistory(true); return false; }
+      })
+    )
+  )
+};
+
+function toggleHistory(keepUIbutton) {
+  var button = $('#toggleHistory');
+
+  portalsHistory.settings.drawMissing = !portalsHistory.settings.drawMissing
+  localStorage[KEY_SETTINGS] = JSON.stringify(portalsHistory.settings);
+  drawAllFlags();
+
+  if (button.hasClass('normHistory')) {
+    button.removeClass('normHistory');
+    button.addClass('revHistory');
+  } else {
+    button.addClass('normHistory');
+    button.removeClass('revHistory');
+  }
+};
+
+function svgToIcon (str, s) {
+  var url = ('data:image/svg+xml,' + encodeURIComponent(str)).replace(/#/g, '%23');
+  return new L.Icon({
+    iconUrl: url,
+    iconSize: [s, s],
+    iconAnchor: [s / 2, s / 2],
+    className: 'no-pointer-events', // allows users to click on portal under the unique marker
```

Comment:
What is 'no-pointer-events'? It is not in your css

Reply:
CSS should be "pointer-events:none"

adding that to CCS-section

Reply:
But do you need it? You already set 'interactive: false'

Reply:
removed